simulation/simple_sensor_simulator/src/vehicle_simulation/ego_entity_simulation.cpp

```diff
@@ -31,6 +31,16 @@ static auto getParameter(const std::string & name, T value = {})
   return value;
 }
 
+template <typename T>
+static auto getParameterFromExistingNode(
+  rclcpp::Node & node, const std::string & name, T value = {})
+{
+  node.declare_parameter<T>(name, value);
+  node.get_parameter<T>(name, value);
+
+  return value;
+}
```

Comment:
Move it into makeSimulationModel (i.e., rewrite it as a function-local lambda expression). In that case, the variable name can be simply `get_parameter`.